src/rgw/rgw_process.cc

```diff
@@ -320,6 +320,10 @@ int process_request(rgw::sal::RGWRadosStore* const store,
 	  << " op status=" << op_ret
 	  << " http_status=" << s->err.http_ret
 	  << " latency=" << s->time_elapsed()
+	  << " op_method=" << s->info.method
+	  << " op_target=" << s->info.request_uri
+          << " op_obj_size=" << s->obj_size
```

Comment:
from the standup today, it sounds like we're still not sure what this part is trying to accomplish. a lot of api calls don't involve objects at all

its probably more useful to just show the request and response sizes, ie:
```c++
      << " received=" << client_io.get_bytes_received()
      << " sent=" << client_io.get_bytes_sent()
```

but all of this info (including method and target) is already present in the frontend log (added by @mkogan1 in https://github.com/ceph/ceph/pull/33083)